tip-wlan/charts/opensync-gw-static/resources/config/logback.xml

```diff
@@ -68,7 +68,7 @@
    -->
    
   <root level="WARN">
-<!--    <appender-ref ref="stdout"/>-->
+    <appender-ref ref="stdout"/>
```

Comment:
Same comment as above (and also for all the changes below). Why are both enabled?